### PR TITLE
Fix moment:20170905 and  static-eval:20171016 vulnerabilities reported by Snyk

### PR DESCRIPTION
**Pull request by matskiv**
Fix for [vulnerability](https://snyk.io/vuln/npm:moment:20170905) reported by [Snyk](https://snyk.io/test/github/evanshortiss/fhlog?severity=high&severity=medium&severity=low).

Changelog: https://github.com/moment/moment/blob/develop/CHANGELOG.md#2193-see-changelog
JIRA: https://issues.jboss.org/browse/FH-4372

### CODE REVIEW

**Comment by matskiv** [comment]
Travis built is failing because of dev dependency. @evanshortiss  can you assess if we can ignore that?

**Comment by matskiv** [comment]
I bumped package version to 0.13.1 because version 0.13.0 is already published on NPM.

**Comment by matskiv** [comment]
We can include fix for [static-eval:20171016](https://snyk.io/vuln/npm:static-eval:20171016) if we wait till [brfs@1.4.4](https://www.npmjs.com/package/brfs) is published. It should be done today as discussed [on this brfs issue](https://github.com/browserify/brfs/issues/81#issuecomment-361544949).

**Comment by matskiv** [comment]
brfs 1.4.4 was just publish on NPM so I pushed a commit with update.

@evanshortiss can you please review and publish on NPM?

File: package.json [hunk 1]
```diff
@@ -40,11 +40,11 @@
   },
   "dependencies": {
     "async": "~0.9.0",
-    "brfs": "~1.2.0",
+    "brfs": "^1.4.4",
```
Comment (evanshortiss) [inline package.json]: @matskiv, would prefer to use `~` instead of `^`.

File: package.json [hunk 2]
```diff
@@ -40,11 +40,11 @@
...
     "colors": "~1.0.3",
     "html5-fs": "0.0.1",
     "lodash": "~2.4.1",
-    "moment": "2.16.0",
+    "moment": "2.19.3",
```
Comment (evanshortiss) [inline package.json]: `~` here also

**Comment by matskiv** [comment]
Changed as requested :)

**Comment by evanshortiss** [comment]
@matskiv last think. I just tested locally and it looks like updating to `"lintspaces-cli": "~0.6.0"` in the package fixes the build.

It also looks like `brfs` is in the package twice. I think you can remove it from `devDependencies`

Thanks

**Comment by matskiv** [comment]
Fixed :)

**Comment by evanshortiss** [comment]
Thank you @matskiv

**Pull request merged**